buildbot/master.cfg

```diff
@@ -347,6 +347,145 @@ def make_fifoci_win(type, mode="normal"):
 
     return f
 
+def make_dolphin_osx_universal_build(mode="normal"):
+    f = BuildFactory()
+
+    f.addStep(GitNoBranch(repourl="https://github.com/dolphin-emu/dolphin.git",
+                          progress=True, mode="incremental"))
+
+    f.addStep(ShellCommand(command=["mkdir", "-p", "build"],
+                           logEnviron=False,
+                           description="mkbuilddir",
+                           descriptionDone="mkbuilddir",
+                           hideStepIf=StepWasSuccessful))
+
+    f.addStep(ShellCommand(command=["../BuildMacOSUniversalBinary.py", "-G", "Ninja"],
+                           workdir="build/build",
+                           description="configuring",
+                           descriptionDone="configure",
+                           haltOnFailure=True))
+
+    f.addStep(Compile(command=["ninja"],
+                      workdir="build/build",
+                      description="building",
+                      descriptionDone="build",
+                      haltOnFailure=True))
+
+    f.addStep(Test(command=["ninja", "unittests"],
+                   workdir="build/build",
+                   description="testing",
+                   descriptionDone="test",
+                   haltOnFailure=True))
+
+    f.addStep(ShellCommand(command="/build/codesign.sh --deep universal/Dolphin.app",
+                           workdir="build/build",
+                           description="signing",
+                           descriptionDone="sign",
+                           haltOnFailure=True))
+
+    f.addStep(ShellCommand(command="rm -rf dmg.dir && mkdir dmg.dir",
+                           workdir="build/build",
+                           description="creating tmp dir",
+                           descriptionDone="create tmp dir",
+                           haltOnFailure=True,
+                           hideStepIf=StepWasSuccessful))
+
+    f.addStep(ShellCommand(command="cp -R universal/Dolphin.app dmg.dir",
+                           workdir="build/build",
+                           description="creating tmp dir",
+                           descriptionDone="create tmp dir",
+                           haltOnFailure=True,
+                           hideStepIf=StepWasSuccessful))
+
+    f.addStep(ShellCommand(command="cp -R universal/Dolphin\ Updater.app dmg.dir",
+                           workdir="build/build",
+                           description="creating tmp dir",
+                           descriptionDone="create tmp dir",
+                           haltOnFailure=True #,hideStepIf=StepWasSuccessful
+                           ))
+
+    f.addStep(ShellCommand(command=r"if [ -d universal/Dolphin\ Updater.app ]; then /build/codesign.sh --deep universal/Dolphin\ Updater.app; cp -R universal/Dolphin\ Updater.app dmg.dir; fi",
```

Comment:
I think that if statement might be unnecessary -- we probably added it to handle the period of time when some PR builds had the updater and some not based on their base revision. Universal builds will always have a base revision that has the auto-updater. WDYT?

Reply:
Yeah, we can just get rid of the check and always sign the updater.